Library/Tracking/KSRAnalytics.swift

```diff
@@ -48,6 +48,7 @@ public final class KSRAnalytics {
     case onboardingContinueButtonClicked = "Onboarding Continue Button Clicked"
     case onboardingGetStartedButtonClicked = "Onboarding Get Started Button Clicked"
     case onboardingSkipButtonClicked = "Onboarding Skip Button Clicked"
+    case pageViewed = "Page Viewed"
```

Comment:
Move this case into a new enum below this one and delete line 51 from here. We are moving toward a more generic solution for event names and will eventually be deprecating this enum one case at a time.